config/initializers/secure_headers.rb

```diff
@@ -8,9 +8,10 @@
       :report_only => false,
       :default_src => ["'self'"],
       :frame_src   => ["'self'"],
-      :font_src    => ["'self'", 'https://fonts.gstatic.com'],
+      :font_src    => ["'self'", 'https://fonts.gstatic.com', "https://fonts.googleapis.com"],
+      :img_src     => ["'self'", "data:"],
       :connect_src => ["'self'"],
-      :style_src   => ["'unsafe-inline'", "'self'"],
+      :style_src => ["'unsafe-inline'", "'self'", "https://fonts.googleapis.com", "https://fonts.gstatic.com"],
```

Comment:
Very minor, but please align the `=>`

Reply:
Should be fixed now if you can review again when you have time